docs/source/reference/ai/model-train-xgboost.rst

```diff
@@ -0,0 +1,26 @@
+.. _xgboost:
+
+Model Training with XGBoost
+============================
+
+1. Installation
+---------------
+
+To use the `Flaml XGBoost AutoML framework <https://microsoft.github.io/FLAML/docs/Examples/AutoML-for-XGBoost/>`_, we need to install the extra Flaml dependency in your EvaDB virtual environment.
+
+.. code-block:: bash
+
+   pip install "flaml[automl] matplotlib openml"
```

Comment:
What is `matplotlib openml` for? I do not install those two dependency in the test, and it seems to work.

Reply:
Yeah not required I guess, removed.